src/renderer/entities/notification/model/notification-model.ts

```diff
@@ -247,6 +253,23 @@ sample({
   target: notificationsFiltered,
 });
 
+sample({
+  clock: notificationsRemoved,
+  source: $notifications,
+  fn: (notifications, { keys }) => {
+    const allThatLeft = notifications.filter((n) => !keys.includes(n.key));
+    const notificationsToRemove = notifications.filter((n) => keys.includes(n.key));
+    return {
+      allThatLeft,
+      notificationsToRemove,
+    };
```

Comment:
do a normal for of loop here please,
```suggestion
    const allThatLeft: Notification[] = [];
    const notificationsToRemove: Notification[] = [];
    
    for (const n of notifications) {
      if (keys.includes(n.key)) {
        notificationsToRemove.push(n);
      } else {
        allThatLeft.push(n);
      }
    }
    
    return {
      allThatLeft,
      notificationsToRemove,
    };
```